.travis.rosinstall.melodic

```diff
@@ -34,3 +34,13 @@
     local-name: cob_supported_robots
     uri: https://github.com/ipa320/cob_supported_robots.git
     version: indigo_dev
+
+
+- git:
+    local-name: realsense_substitute
+    uri: https://github.com/ipa320/realsense.git
+    version: realsense_substitute
+- git:
+    local-name: realsense2_substitute
+    uri: https://github.com/ipa320/realsense.git
+    version: realsense2_substitute
```

Comment:
this is needed because `realsense2_camera` is not released into `melodic`
(although, I commented that dependency again)